src/libraries/System.Private.CoreLib/src/System/ArgumentOutOfRangeException.cs

```diff
@@ -22,31 +22,22 @@ public class ArgumentOutOfRangeException : ArgumentException
         // Creates a new ArgumentOutOfRangeException with its message
         // string set to a default message explaining an argument was out of range.
         public ArgumentOutOfRangeException()
-            : base(SR.Arg_ArgumentOutOfRangeException)
-        {
-            HResult = HResults.COR_E_ARGUMENTOUTOFRANGE;
-        }
+            : this(null, null, null) {}
 
         public ArgumentOutOfRangeException(string? paramName)
-            : base(SR.Arg_ArgumentOutOfRangeException, paramName)
-        {
-            HResult = HResults.COR_E_ARGUMENTOUTOFRANGE;
-        }
-
+            : this(paramName, null, null) {}
+ 
```

Comment:
```suggestion

```

Reply:
I can't really see any trailing whitespace here. Do you mean no empty line between members?

Reply:
No, I mean remove the single space that was on the line

Reply:
I think this was fixed by https://github.com/dotnet/runtime/pull/89846/commits/3f624ff6fc8e347efde954744879c075efb1411b

Reply:
It was not. It's failing CI because the whitespace is still there.

Reply:
oh, ok.
Seems I need to checkout locally